module/documents/actor/actor.mjs

```diff
@@ -277,6 +277,19 @@ export default class Actor5e extends SystemDocumentMixin(Actor) {
 
   /* -------------------------------------------- */
 
+  /**
+   * Calculate the bonus from any cover the actor is affected by.
+   * @returns {number}      The cover bonus to AC and dexterity saving throws.
+   */
+  getCoverBonus() {
```

Comment:
Since we aren't doing anything complex here, it is probably better as just a standard getter up in the properties section.

```suggestion
  get coverBonus() {
```